src/NetscriptFunctions/StockMarket.ts

```diff
@@ -307,6 +307,10 @@ export function NetscriptStockMarket(): InternalAPI<TIX> {
       return true;
     },
     purchaseTixApi: (ctx) => () => {
+      if (Player.bitNodeOptions.disable4SData) {
```

Comment:
The fact that this wasn't here before is evidence that TIX access wasn't *meant* to be disabled.